README.md

```diff
@@ -26,6 +26,17 @@ plus common / utility classes used across article elements
 
 ## Migration guides
 
+### v9 to v10
+
+v10 moves the responsibility of setting the width of embedded YouTube videos to the consumer, in relation to the 'n-content-video--youtube' class.
```

Comment:
Now that we've confirmed that the decisions decision is to have full width videos across FT.com should we consider re-wording this so people know they don't need to make a change.

We can keep the steps below if they do need to bring back the previous behaviour for some reason.

Reply:
Good point, looking into it now.